src/views/Batches/batchComponents/BatchForm.jsx

```diff
@@ -234,6 +234,12 @@ const BatchForm = (props) => {
     });
   }
 
+  const paymentOptions = [
+    { value: "Free", label: "Free" },
+    { value: "Paid by College", label: "Paid by College" },
+    { value: "Paid by Students", label: "Paid by Students" }
+  ];
```

Comment:
we're using the `Picklist` component in Strapi to define all dropdown field options. Let's move these there and make an API call to get these options.

Please refer to the picklist field config in your Strapi and see implementation of any of these fields on the frontend.
<img width="1523" alt="image" src="https://user-images.githubusercontent.com/12053186/231387856-f232b479-61f7-4a41-b66b-a7638b413997.png">